assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -583,8 +583,6 @@ describe( 'AudienceTilesWidget', () => {
 
 		await waitForRegistry();
 
-		expect( mockTrackEvent ).not.toHaveBeenCalled();
-
```

Comment:
We shouldn't need to remove this assertion - rather, we need to ensure the gathering data report is correctly mocked for this test. I've pushed a fix for this to save a round trip.